test/routes/setup_test.js

```diff
@@ -0,0 +1,18 @@
+let path = require('path');
```

Comment:
Can we move this to `test/setup.js` since it doesn't actually have any tests? (just bootstrapping the thing)